### DISCUSSION TITLE
Trail on mousemove only not on hover

### DISCUSSION THREAD

**Discussion by masteryder** [matteobruni/tsparticles#1491] [Q&A] [answered]
Hi, I'm trying out ts particles for the first time and am wondering if there is a way of having a trail appear only when the mouse is moving, instead of on hover?

**Comment by matteobruni** [comment, marked as answer]
Hi @masteryder,

Thank you for this, I forgot this feature in the v2 which is under development but it can be easily brought to v1 as well. It will be available in some days in the new minor version.

You can pause the trail using this property

```json
"pauseOnStop": true,
``` 

inside the `interactivity.modes.trail` options
